test/integration/fork_choice/handlers_test.exs

```diff
@@ -20,9 +25,58 @@ defmodule Integration.ForkChoice.HandlersTest do
     {:ok, signed_block} = BlockStore.get_block_by_slot(state.slot)
     {:ok, new_signed_block} = BlockStore.get_block_by_slot(state.slot + 1)
 
-    assert {:ok, store} = Types.Store.get_forkchoice_store(state, signed_block.message, true)
+    assert {:ok, store} = Types.Store.get_forkchoice_store(state, signed_block, true)
     new_store = Handlers.on_tick(store, :os.system_time(:second))
 
     assert {:ok, _} = Handlers.on_block(new_store, new_signed_block)
   end
+
+  @tag :skip
```

Comment:
Not sure about the usefulness of this test. Shouldn't we prefill the db with some fixtures?

Reply:
It's meant as a benchmark, to quickly find troublesome blocks in a range. I moved it to `bench/`